app/components/dashboard-myreports.js

```diff
@@ -35,12 +34,9 @@ export default Component.extend(DomMixin, ReportTitleMixin, {
    * @type {Ember.computed}
    * @public
    */
-  sortedReports: computed('user.reports.[]', function() {
-    return new Promise(resolve => {
-      this.user.get('reports').then(reports => {
-        resolve(reports.sortBy('title'));
-      });
-    });
+  sortedReports: computed('user.reports.[]', async function() {
+    const reports = await this.user.reports;
```

Comment:
I don't think you can chain getters in ember-data yet (according to something I saw somewhere i don't remember) so this should be `await this.user.get('reports');

Reply:
I think I checked all updates and they were coming back the same.

```
// From
something.get('something');

// To
something.something
```

Reply:
Reverted and addressed.